resolwe/elastic/viewsets.py

```diff
@@ -0,0 +1,69 @@
+""".. Ignore pydocstyle D400.
+
+================
+Elastic Viewsets
+================
+
+.. autoclass:: resolwe.elastic.viewsets.ElasticSearchViewSet
+    :members:
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from elasticsearch_dsl.query import Q
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from rest_framework.viewsets import GenericViewSet
+
+
+__all__ = ('ElasticSearchViewSet',)
+
+
+class ElasticSearchViewSet(GenericViewSet):
```

Comment:
Should this viewset also include methods for handling most common search requests, so that the user needs to only override a few specific things and common stuff like ordering and pagination are handled correctly? An even better approach would be to maybe have this as a mixin.